cozy.example.yaml

```diff
@@ -205,6 +205,7 @@ redis:
     konnectors: 5
     realtime: 6
     log: 7
+    rate_limiting: 8
```

Comment:
@sblaisot we will need to update the config file for this pull-request

Reply:
no problem in our infra.
ping @aeris for self-hosted